doc/dev/release.rst

```diff
@@ -216,7 +214,7 @@ so:
         virtualenv build
         cd build
         source bin/activate
-        pip install -U setuptools==3.4.1 wheel
+        pip install wheel
```

Comment:
Good point.

(My gentoo heritage is screaming: "Damn it, the correct version of everything is the cutting edge version." :) )